polynote-kernel/src/test/scala/polynote/kernel/remote/RemoteKernelSpec.scala

```diff
@@ -125,24 +125,24 @@ abstract class RemoteKernelSpecBase extends FreeSpec with Matchers with ZIOSpec
       }
 
       "handles notebook updates" in {
```

Comment:
Oops, I should re-add this test. I had to get rid of some sketchy machinery that made it possible.